src/id128.rs

```diff
@@ -46,3 +46,21 @@ impl Id128 {
         &self.inner.bytes
     }
 }
+
+impl From<ffi::id128::sd_id128_t> for Id128 {
```

Comment:
This exposes a type from a different crate in the public API (which is generally problematic for semver). Also, I think that the idea is not to expose unsafe FFI bits in the safe high-level API.
I think a better approach would be to keep this as a private method. Or to implement fallible conversion from the two slice forms and with UUID semantic validation.

Reply:
I've changed this to a crate-level `from_ffi()` instead.